src/ComboBox/index.jsx

```diff
@@ -447,7 +466,6 @@ export default class ComboBox extends Component
             isMultiselect,
             isReadOnly,
             isSearchable,
-            options,
```

Comment:
This shouldn't be removed, "normalized options" should be passed directly from `getDerivedStateFromProps()`